src/components/project/home-block.vue

```diff
@@ -0,0 +1,68 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="project-home-block">
+    <h2 class="title">
+      <router-link :to="projectPath(project.id)">{{ project.name }}</router-link>
+    </h2>
+    <table v-if="visibleForms != null" class="project-form-table table">
+      <project-form-row v-for="form of visibleForms" :key="form.xmlFormId" :form="form" :columns="columns"/>
+    </table>
+  </div>
+</template>
+
+<script>
+import Project from '../../presenters/project';
+import Form from '../../presenters/form';
+import routes from '../../mixins/routes';
+
+import ProjectFormRow from './form-row.vue';
+
+export default {
+  name: 'ProjectHomeBlock',
+  components: { ProjectFormRow },
+  mixins: [routes()],
+  props: {
+    project: {
+      type: Project,
+      required: true
+    },
+    maxForms: {
+      type: Number,
+      default: 3
+    }
+    // todo: default # of proj to show
+  },
+  computed: {
+    visibleForms() {
+      return this.project.nestedForms.slice(0, this.maxForms).map((f) => new Form(f));
```

Comment:
Instead of wrapping the forms in a presenter here, I think it'd be better to do it when the response is received (in the store). That would involve changing the `projects` transform function from:

```js
({ data }) => data.map(project => new Project(project))
```

to something like:

```js
({ data }) => data.map(project => new Project({
  ...project,
  forms: forms.map(form => new Form(form))
}))
```